x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.scss

```
@@ -1,3 +1,6 @@
 .alertsTableActiveRow {
   background-color: tintOrShade($euiColorLightShade, 50%, 0);
 }
+.euiDataGridRowCell {
+  font-size: $euiFontSizeXS;
+}
```

This will end up affecting other data grid tables across the entire Kibana app. You should nest this in a CSS class that only affects your desired data grid, e.g.

```suggestion
.alertsTable .euiDataGridRowCell {
  // This is a temporary CSS workaround while figuring out why `fontSize: 's'`
  // causes broken rendering on the data grid's style settings
  font-size: $euiFontSizeXS;
}
```

Not sure if you actually have the `alertsTable` class on your data grid already, if not you'll need to add it.

I also added a comment noting that this is more of a workaround for now, and when it would be safe to remove

---

fix with 97e5689cfa63a3902dac63c1947ea808d6860409

---

Thanks, that addresses the specificity issue. I'd consider adding a comment still for you/other devs to know when the workaround CSS can be removed, but that's less blocking :)